fire/SFMainMod.F90

```diff
@@ -916,21 +923,49 @@ subroutine  crown_damage ( currentSite )
+                ! Calculate clear branch bole height at base of crown
+                ! inst%crown = crown_depth_frac (PFT)
+                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   currentCohort%passive_crown_FI = (0.01 * height_cbb * SF_val_crown_ignition_energy)**1.5
+
+                   if (currentCohort%passive_crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 14 Bessie and Johnson 1995
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%passive_crown_FI
+                      
+                      if (currentCohort%ignite_crown > 1.0_r8) then
+                         currentCohort%crown_fire_flg = 1  ! passive crown fire ignited
+                         currentCohort%fraction_crown_burned =  1.0_r8
+                      ! else ! evaluate crown damage based on scorch height
+                      endif ! ignite passive crown fire
+                   ! else no crown fire today
+                   endif ! crown fire intensity
+                ! else ! not crown fire plant
+                endif ! evaluate passive crown fire
+                
                 ! Flames lower than bottom of canopy. 
-                ! c%hite is height of cohort
-                if (currentPatch%SH < (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft))) then 
+                ! c%hite is height of cohort  
+                if (currentPatch%SH < (currentCohort%hite- height_cbb)) .and. currentCohort%crown_fire_flg = 0 then 
                    currentCohort%fraction_crown_burned = 0.0_r8
                 else
                    ! Flames part of way up canopy. 
                    ! Equation 17 in Thonicke et al. 2010. 
                    ! flames over bottom of canopy but not over top.
                    if ((currentCohort%hite > 0.0_r8).and.(currentPatch%SH >=  &
-                        (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)))) then 
+                      (currentCohort%hite - height_cbb))) .and. currentCohort%crown_fire_flg = 0 then 
 
-                           currentCohort%fraction_crown_burned =  (currentPatch%SH-currentCohort%hite*(1.0_r8- &
-                                EDPftvarcon_inst%crown(currentCohort%pft)))/(currentCohort%hite* &
-                                EDPftvarcon_inst%crown(currentCohort%pft)) 
+                      currentCohort%fraction_crown_burned =  (currentPatch%SH-currentCohort%hite*(1.0_r8- &
+                             EDPftvarcon_inst%crown(currentCohort%pft)))/(height_cbb) 
```

Comment:
...while in this line you'd say:

```suggestion
                             / (currentCohort%hite * EDPftvarcon_inst%crown(currentCohort%pft))
```

Reply:
@jkshuman pls let me know if anything I'm suggesting is unclear and also if you think that I have misunderstood.

Reply:
@slevisconsulting Thanks for checking this over. I agree with your assessment, and I fixed this based on your suggestions. Please let me know if you think it looks good now.